tests/integration/serverless.yml

```diff
@@ -20,6 +20,21 @@ provider:
   iam:
     # IAM permissions require that all functions are deployed with this role
     role: "arn:aws:iam::425362996713:role/serverless-integration-test-lambda-role"
+  layers:
+    - { Ref: PythonLambdaLayer }
+    - { Ref: PythonRequirementsLambdaLayer }
+
+plugins:
+  - serverless-python-requirements
+
+custom:
+  pythonRequirements:
+    pythonBin: python3
+    pipCmdExtraArgs:
+      - --no-deps  # install just requests
+    layer:
+      compatibleRuntimes:
+        - ${env:SERVERLESS_RUNTIME}
```

Comment:
Why do we need to add new requirements? Could you give me a brief explanation so I understand it?

Reply:
We removed `requests` from the lambda layer, yet the tests still rely on it. So we must add `requests` back.

Reply:
Thanks! I imagine these are the ones that ping URLs?